### PR TITLE
Concourse Is Ready to Live In Master

### PR DESCRIPTION
**Pull request by zaksoup**
This pull request pulls in the configuration and scripts for concourse to the master branch of the repo.

## Changes Made
* a new folder has been added: `concourse/`
* a README.md section has been added for "experimental concourse support"

**Note:** the concourse pipelines have not been switched to the `master branch`. Concourse pipelines will continue to target the `concourse-infrastructure` branch for faster iteration.

### CODE REVIEW

File: concourse/package_tarball_debian8_debug.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+platform: linux
+image: docker:///yolo/orcadev#jessie
```
Comment (hsyuan) [inline concourse/package_tarball_debian8_debug.yml]: is it better to rename jessie to debian8?

Comment (vraghavan78) [inline concourse/package_tarball_debian8_debug.yml]: why are we using yolo docker image?

**Comment by vraghavan78** [comment]
What if hypothetically yolo decides to nuke that docker image?

**Comment by zaksoup** [comment]
@vraghavan78 we're planning on building our own docker image in this CI. there are stories in the future for this, so I'm not worried about yolo pulling the image in the next few weeks.

**Comment by vraghavan78** [comment]
LGTM

**Comment by zaksoup** [comment]
@vraghavan78 do I need another +1 or can I push the merge button?

**Comment by vraghavan78** [comment]
Please go ahead and merge.

**Pull request merged**